cores/esp8266/Updater.cpp

```diff
@@ -224,71 +227,87 @@ bool UpdaterClass::end(bool evenIfRemaining){
   }
 
   if (_verify) {
+    // If expectedSigLen is non-zero, we expect the last four bytes of the buffer to
+    // contain a matching length field, preceded by the bytes of the signature itself.
+    // But if expectedSigLen is zero, we expect neither a signature nor a length field;
+    static constexpr uint32_t sigSize = 4;
```

Comment:
`sigSize = 4` was `sigSize = sizeof(uint32_t)` or better should be `sigSize = sizeof(sigLen)` no ?
(I must mention it, because I'd feel bad to go on with teaching students that constants are forbidden in the middle of a source code)

Reply:
It was, just all over the place. `using SigType = uint32_t;` could also work? Changed to sizeof right now
(but... scope constants are extremely useful, I guess am a rebel like that)

Reply:
I guess anything but a scalar would do it :)